### PR TITLE
Add Community section, including events and chat

### PR DESCRIPTION
**Pull request by cmrnh**
Following up on #305 — let me know what you think @sindresorhus!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -665,6 +665,17 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [generator-nm](https://github.com/sindresorhus/generator-nm) - Scaffold out a node module.
 
 
+## Community
+
+### Events
+
+- [Live help hangouts](http://start.thinkful.com/node/) - Bring questions and pair with other node developers in a live group hangout. Hear about the challenges other developers are running into, or screenshare your own code with the group for feedback.
+
+### Chat rooms
```
Comment (sindresorhus) [inline readme.md]: Maybe `Discussions` to allow for forums too?

File: readme.md [hunk 2]
```diff
@@ -665,6 +665,17 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
...
+
+- [nodeslackers.io](http://nodeslackers.io) - A real-time discussion place for literally anything Node.js with over 500 members.
```
Comment (sindresorhus) [inline readme.md]: Ugh, Slack really...

https://news.ycombinator.com/item?id=9754626

**Comment by cmrnh** [comment]
Oh, forgot to reply re: Slack — can remove it? I'm on that Node.js Slack and Reactiflux, and I've found them both to be helpful. Any other discussion-oriented resources you think we should add?

**Comment by sindresorhus** [comment]
> Slack — can remove it?

I wanted to, but thinking about it a bit, there's already a lot of people there, so it's an useful resource, even though I strongly disagree with how it works. So keep it for now.

> Any other discussion-oriented resources you think we should add?

http://stackoverflow.com/questions/tagged/node.js and IRC irc.freenode.net #node.js

**Comment by sindresorhus** [comment]
ping @cmrnh :)

**Pull request closed without merging**